scripts/updateContributors.mjs

```diff
@@ -0,0 +1,79 @@
+import simpleGit from 'simple-git';
+import { Octokit } from '@octokit/rest';
+import fs from 'node:fs';
+import path from 'path';
+import pMemoize from 'p-memoize';
+import LOST_CONTRIBUTORS from './lost-contributors.json' assert { type: 'json' };
```

Comment:
This is lost as in?... Not carried over from Feather?
If so, maybe let's call this `feather-contributors.json` or `original-contributors.json`.

Reply:
Mostly for feather, but also if you mess up the history in some other way.